spec/unit/app/controllers/concerns/scheduling_validator_spec.rb

```diff
@@ -29,16 +27,14 @@ def scheduled_publication_params
   end
 end
 
-class EditionFormTest < ActiveSupport::TestCase
-  extend Minitest::Spec::DSL
-
+RSpec.describe EditionForm do
   def it_raises_a_validation_error(object, error_key)
     error = assert_raises ActiveRecord::RecordInvalid do
       object.validate_scheduled_edition
     end
 
-    assert_equal 1, error.record.errors.messages.count
-    assert_equal I18n.t("activerecord.errors.models.edition.attributes.#{error_key}"), error.record.errors.full_messages[0]
+    expect(error.record.errors.messages.count).to eq(1)
+    expect(error.record.errors.full_messages.[](0)).to eq(I18n.t("activerecord.errors.models.edition.attributes.#{error_key}"))
```

Comment:
this method of accessing a list looks odd: `error.record.errors.full_messages.[](0)` ?

How about `error.record.errors.full_messages.first`?

Reply:
Oh yeah this is quite odd isn't it. Will adjust.